### ISSUE TITLE
Where are the Dart communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/dart#125]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Dart supported by one or more large organizations?
* Does Dart have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Dart? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Dart? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Dart? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Dart?
* Are there popular newsletters for Dart?
* Is Dart taught at programming bootcamps? (If so, what are some examples?)
* Is Dart taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Dart?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by Stargator** [comment]
* Is Dart supported by one or more large organizations?
  - **Dart is supported by Google**

* Does Dart have an official community manager?
  - **Again, Google but more specifically the Dart Team.**

* Do you know of specific communities (online or offline) that are enthusiastic about Dart? (Chat communities, forums, meetups, student clubs, etc)
  - **I can only provide you with links found at https://www.dartlang.org/community**
  - **Several channels on Gitter: https://gitter.im/dart-lang/home**
  - **Reddit - https://www.reddit.com/r/dartlang**
  - **Google Plus - http://g.co/dartisans**
  - **General Discussion on Google Groups - https://groups.google.com/a/dartlang.org/d/forum/misc**
  - A local meetup in Northern Virginia (MoDev) is going to have a event on June 12th for Flutter, a Dart-based framework for mobile app development. If anyone is interested in Flutter for cross-platform apps (iOS and Android): https://www.meetup.com/modevdc/events/251251800

Flutter is a Dart-based framework for mobile app development.

* Are there popular conferences for Dart? (If so, what are some examples?)
  - **DartConf - https://news.dartlang.org/2017/11/you-are-invited-to-dartconf-los-angeles.html**

* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Dart? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

* Are there specific groups or programs dedicated to mentoring people in Dart?
  - **Not that I know of.**

* Are there popular newsletters for Dart?
  - **Not anymore, it stopped in November 2017: https://github.com/dart-lang/sdk/blob/master/docs/newsletter/README.md#dart-language-and-library-newsletters**

* Is Dart taught at programming bootcamps? (If so, what are some examples?)
  - **Not at programming bootcamps, but the community has the Dart Academy website that still seems action: https://dart.academy/**

* Is Dart taught at universities? (If so, what are some examples?)
  - **Not that I know of**

In other words: where do people care a lot and/or know a lot about Dart?
  - The Google Groups and the Gitter channels linked above

**Comment by kytrinyx** [comment]
This is fantastic, thank you!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. (Should you come across any other good sources, feel free to add them here, I'll be coming back to this). ❤️ 💛 💚 💙 💜 🖤 Thanks!

**Comment by Stargator** [comment]
@kytrinyx just came here to say that dev.to had a few Dart interested folks. Just look up the `dart` tag.

That's how I got @jvarness to help us out

**Comment by kytrinyx** [comment]
Oh, fabulous!

**Comment by Stargator** [comment]
Dev.to also regularly posts a [calling for any projects in need of contributors](https://dev.to/search?q=Open+Source+contributors+June)
